### PR TITLE
feat(esx_skin): update player weight on spawn

### PR DESCRIPTION
**Pull request by SirLamaGott**
This updates the max weight on join, if a player has a bag.

### CODE REVIEW

**Comment by TrymTube** [comment]
Snippet made by me, PR posted by @SirLamaGott

**Review by TrymTube — APPROVED** [review]

File: [esx]/esx_skin/client/main.lua [hunk 1]
```diff
@@ -1,6 +1,13 @@
 local lastSkin, cam, isCameraActive
 local firstSpawn, zoomOffset, camOffset, heading, skinLoaded = true, 0.0, 0.0, 90.0, false
 
+RegisterNetEvent('esx:playerLoaded')
+AddEventHandler('esx:playerLoaded', function(xPlayer, isNew, skin)
+    if ESX.IsPlayerLoaded() then
```
Comment (Gellipapa) [inline [esx]/esx_skin/client/main.lua:6]: @SirLamaGott Remove please this if statement, i think don't need check player is loaded if we call event in esx:playerLoaded event. 

I tested and i get false ESX.IsPlayerLoaded() all time.

Comment (SirLamaGott) [reply [esx]/esx_skin/client/main.lua]: I removed it.

**Review by Gellipapa — APPROVED** [review]
Okey thank you. @SirLamaGott I merged.

**Review by bitpredator — APPROVED** [review]

**Pull request merged**